src/js/service/ImportService.js

```diff
@@ -80,6 +80,73 @@
   };
 
   /**
+   * Given an image object and some options, merge the frames with the currently open
+   * Piskel and open it for editing.
+   * @param {!Image} image
+   * @param {!Object} options
+   * @param {!string} options.importType - 'single' if not spritesheet
+   * @param {!number} options.frameSizeX
+   * @param {!number} options.frameSizeY
+   * @param {number} [options.frameOffsetX] only used in spritesheet imports.
+   * @param {number} [options.frameOffsetY] only used in spritesheet imports.
+   * @param {!boolean} options.smoothing
+   * @param {number} [options.frameRate] in frames per second
+   * @param {function} [onComplete]
+   */
+  ns.ImportService.prototype.importFramesFromImage = function (image, options, onComplete) {
+    onComplete = onComplete || Constants.EMPTY_FUNCTION;
```

Comment:
This logic is mostly borrowed from `ns.ImportService.prototype.newPiskelFromImage`
Wasn't sure, since we try not to make piskel too hard to upgrade in the future, if I should refactor out some of the logic, or just copy and add my own new function. In the past I've been trying to only add code to piskel, in hopes that merging might be easier.